sdk/digitaltwins/Azure.DigitalTwins.Core/tests/QueryBuilderTests.cs

```diff
@@ -0,0 +1,103 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This is great, I like the QueryBuilderTests. 
You also can start testing individual components (SelectQuery, WhereQuery, FromQuery) on their own. let me know if you have any questions.